nixos/modules/installer/tools/nixos-version.sh

```diff
@@ -13,6 +13,13 @@ case "$1" in
     fi
     echo "@revision@"
     ;;
+  --configurationRevision)
```

Comment:
```suggestion
  --configuration-revision)
```
flags are usually this style IIRC.